### DISCUSSION TITLE
GitHub accounts: 4ian vs. GDevelopApp

### DISCUSSION THREAD

**Discussion by ghost** [4ian/GDevelop#4177] [Technical questions about the codebase]
I'm sure this has likely been discussed previously, but I couldn't find anything about it...

I'm wondering what, if any, considerations have been given to moving the GDevelop repo (https://github.com/4ian/GDevelop) from @4ian's account into the GDevelopApp one (https://github.com/GDevelopApp)?

The reason I ask is because I often come here looking for examples or extensions, but then I remember that they're in a different repo, and I eventually find them under https://github.com/GDevelopApp/GDevelop-examples and https://github.com/GDevelopApp/GDevelop-extensions respectively.

Then I wonder why this repo doesn't live over there as well (to keep everything in the same place)...

**Comment by arthuro555** [comment]
I'm guessing it's to avoid broken links, since this has been GDevelop's repository for... almost 10 years I think? Where the GDevelopApp account is pretty recent.

**Reply by Bouh** [reply]
I think this is the main reason.
It would break a lot of links that depend on it, and not only internal links with the application we can fix them.
I'm thinking of links to press articles, but also to directory sites or even package sites.

**Comment by 4ian** [comment]
In theory we would have everything under GDevelopApp, but I was a bit worried of breaking things. I know that GitHub does redirections though, so in theory we could move this to GDevelopApp/GDevelop. I guess we'll have to do this one day! :)
